translations/README.fr.md

```diff
@@ -16,7 +16,15 @@ Azure Cloud Advocates de Microsoft ont le plaisir de vous proposer un programme
 
 **Un grand merci à nos auteurs Jen Looper, Chris Noring, Christopher Harrison, Jasmine Greenaway, Yohan Lasorsa, Floor Drees et l’artiste de sketchnote Tomomi Imura!**
 
-# Pour commencerhttps://ashy-river-0debb7803.1.azurestaticapps.net/
+# êtes vous étudiant ?
```

Comment:
Translations should match the english content, and this section does not exist in the original page. Could you remove it so we can merge your other fixes?

Reply:
i'm pretty sure that this section exist in english
https://github.com/microsoft/Web-Dev-For-Beginners

Reply:
My bad, I don't know how I missed it!